pages/1.14/administering-clusters/draining-a-node/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle:  Draining a Node
+title: Draining a Node
+menuWeight: 801
+excerpt: Draining an agent node in an active DC/OS cluster
+enterprise: false
+render: mustache
+model: /1.14/data.yml
+---
+
+You can drain tasks from an agent node in an active DC/OS cluster in order to take the node down for maintenance or to remove it permanently from the cluster. Nodes may be drained via the DC/OS CLI, and progress can be monitored via the CLI or UI.
+
+<p class="message--warning"><strong>WARNING: </strong>All tasks that are running on the agent will be killed when draining is initiated. Use this feature with caution!</p>
```

Comment:
We should also mention here that if `--decommission` is used then any local persistent data will be lost.